packages/office-ui-fabric-react/src/components/DetailsList/DetailsList.base.tsx

```diff
@@ -103,21 +106,12 @@ export class DetailsListBase extends BaseComponent<IDetailsListProps, IDetailsLi
   constructor(props: IDetailsListProps) {
     super(props);
 
+    initializeComponentRef(this);
+    initializeFocusRects();
```

Comment:
Is this the only component where we need to initialize focus rects?

Reply:
The initialization only *needs* to be done once on the whole page (though BaseComponent did it in all subclasses). So it seemed best for perf to only do it in a root component.